### DISCUSSION TITLE
Multiple inline image (base64) becomes laggy

### DISCUSSION THREAD

**Discussion by DreAmigo** [tinymce/tinymce#9511] [General]
I am using TinyMCE v7 with React, and I have a few inline images coming through the `initialValue()`. With that said, the editor becomes very laggy while I enter or type anything into it.

What is the best approach to work around this issue? We don't want to upload the images to any server and it will be used as an email editor.

Any ideas will be grateful appreciated :)

**Comment by GrtFS** [comment]
I use Tiny in a FileMaker Pro database, where it runs **inline** in a 'web-viewer'.

I found in TinyMCE 5 and 6, records with just a few base 64 images become agonisingly slow to load the page.  So I've reduced the size of the images to the size that is actually being displayed and this helped a LOT.  However, it doesn't fix the problem.  I haven't checked with TinyMCE 7, to see if there are improvements.  From what you say, there clearly aren't.
